src/index.test.ts

```diff
@@ -151,11 +151,12 @@ describe('DynamoDBCache', () => {
 
   describe('basic functionality for tables with compound keys', () => {
     describe('get', () => {
-      it.skip('can retrieve an existing key', async () => {
```

Comment:
Why were these skipped?

Reply:
I implemented this test in https://github.com/rbilabs/apollo-server-cache-dynamodb/pull/1, but it's skipped there because it's failing (and expected to fail). I re-enable the test here since these changes fix the problem.